test.rb

```diff
@@ -0,0 +1,36 @@
+a = "bbb"
+
+def foo
+    'a'
+end
+
+def a
+  def b
+    1
+    ['a','c']
+    'C'
+    "A"
```

Comment:
__[development][rubocop]__
* Literal `'C'` used in void context.
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->